overlays/wwinit/init

```diff
@@ -26,31 +26,38 @@ mount -t tmpfs tmpfs /run
 
 chmod 755 /warewulf/wwinit
 
-echo "Checking Rootfs type"
+echo -n "Checking Rootfs type..."
 ROOTFSTYPE=`stat -f -c "%T" /`
+echo "${ROOTFSTYPE}"
 
-if test "$WWROOT" = "initramfs"; then
-    echo "Provisioned to default initramfs file system: $ROOTFSTYPE"
-    echo "Calling WW Init"
-    exec /warewulf/wwinit
-elif test "$WWROOT" = "tmpfs"; then
-    if test "$ROOTFSTYPE" = "tmpfs"; then
-        echo "ERROR: Switching the root file system requires the kernel argument: 'rootfstype=ramfs'"
-    else
-        echo "Setting up tmpfs root file system"
+
+if [ -n "${WWROOT}" ] && [ "${WWROOT}" != "${ROOTFSTYPE}" ]; then
+    echo "Requested rootfs type does not match current rootfs type. Pivoting to new rootfs..."
+    if [ "${WWROOT}" == "ramfs" ] || [ "${WWROOT}" == "tmpfs" ]; then
         mkdir /newroot
-        mount wwroot /newroot -t tmpfs
-        echo "Moving RAMFS to TMPFS"
-        tar -cf - --exclude ./proc --exclude ./sys --exclude ./dev --exclude ./newroot . | tar -xf - -C /newroot
-        mkdir /newroot/proc /newroot/dev /newroot/sys /newroot/run 2>/dev/null
-        echo "Calling switch_root and invoking WW Init"
+        mount wwroot /newroot -t ${WWROOT}
+        tar cf - --exclude ./proc --exclude ./sys --exclude ./dev --exclude ./run --exclude ./newroot . | tar xf - -C /newroot
+        mkdir /newroot/proc /newroot/sys /newroot/dev /newroot/run 2>/dev/null
+        echo "Switching to new rootfs and invoking /warewulf/wwinit..."
         exec /sbin/switch_root /newroot /warewulf/wwinit
+    elif [ "${WWROOT}" == "initramfs" ]; then
+        echo ">>> Deprecated option \"initramfs\" chosen."
+        echo ">>> The 'initramfs' can be either a ramfs or a tmpfs natively, depending on kernel configuration."
+        echo ">>> If you wish to use whatever this kernel chooses by default, remove the 'root' setting from your nodes/profiles."
+        echo "Invoking /warewulf/wwinit..."
+        exec /warewulf/wwinit
+    else
+        echo "Unrecognized rootfs type requested: ${WWROOT}"
+        echo "Rebooting in 1 minute..."
+        sleep 60
+        /sbin/reboot -f
     fi
 else
-    echo "ERROR: Unknown Warewulf Root file system: $WWROOT"
+    echo "Invoking /warewulf/wwinit..."
+    exec /warewulf/wwinit
 fi
 
 echo
-echo "There was a problem with the provisioning process, rebooting in 1 minute..."
+echo "There was a problem with the initial provisioning process, rebooting in 1 minute..."
 sleep 60
-/sbin/reboot
\ No newline at end of file
+/sbin/reboot -f
```

Comment:
`/sbin/reboot` is a softlink to `systemctl` and may working without `systemd` running, or does the `-f` switch override this. @elfurbe can you please test this? 
An alternative is `echo b > /proc/sysrq-trigger` which always works.